x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -154,27 +175,17 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
         const timestamp = startedAt.toISOString();
         const actionGroupId = FIRED_ACTIONS_ID; // Change this to an Error action group when able
         const reason = buildInvalidQueryAlertReason(params.filterQueryText);
-        const alert = alertFactory(UNGROUPED_FACTORY_KEY, reason, actionGroupId);
-        const alertUuid = getAlertUuid(UNGROUPED_FACTORY_KEY);
-        const indexedStartedAt =
-          getAlertStartedDate(UNGROUPED_FACTORY_KEY) ?? startedAt.toISOString();
-
-        alert.scheduleActions(actionGroupId, {
```

Comment:
It is rolled into the `alertsClient.report` call. It doesn't seem like any rule executor reports an alert without wanting an action scheduled for it, so we do it in a single call instead of asking rule executors to explicitly create and then schedule.